docs/layers.md

```diff
@@ -796,6 +796,7 @@ To improve performance, some road segments are merged at low and mid-zooms. To f
 * `aeroway`: See kind list below.
 * `ferry`: See kind list below.
 * `highway`: See kind list below.
+* `hiking_network`: Present if the road is part of a hiking network. If so, the value will be one of `iwn` for International Walking Network, `nwn` for National Walking Network, `rwn` for Regional Walking Network, `lwn` for Local Walking Network.
```

Comment:
I went back and forth on this. On the one hand, it's `route=hiking`, on the other it's `*wn` for Walking Network. Whatever you think is more consistent and easy to understand - it's only a property name.